File: Course_01/C++ Language/EstruturaSequencial/terreno.cpp
#include <iostream>

using namespace std;

int main()
{
    double preco, area, valorMetro, largura, comprimento;

    cout << "Digite a  largura do terreno: ";
    cin >> largura;
    cout << "Digite o comprimento do terreno: ";
    cin >> comprimento;
    cout << "Digite o valor do metro quadrado: ";
    cin >> valorMetro;

    area = largura * comprimento;

    preco = area * valorMetro;

    cout << "Area do terreno = " << area << endl;
    cout << "Preco do terreno = " << preco << endl;

    return 0;
}

File: Course_01/Python/EstruturaSequencial/terreno.py
largura = float(input("Digite a largura do terreno: "))
comprimento = float(input("Digite o comprimento do terreno: "))
valorMetro = float(input("Digite o valor do metro quadrado: "))

area = largura * comprimento

preco = area * valorMetro

print(f"Área do terreno = {area}")
print(f"Preço do terreno = {preco}")